main.go

```diff
@@ -26,6 +27,8 @@ func main() {
 		logr.Infof("Delaying start by %d seconds", delayInSeconds)
 		time.Sleep(time.Duration(delayInSeconds) * time.Second)
 	}
+	buildInfo := buildinfo.Get()
+	logr.Infof("Starting Gatus (version: %s, revision: %s, build-date: %s)", buildInfo.Version, buildInfo.Revision, buildInfo.Date)
```

Comment:
```suggestion
	logr.Infof("Starting Gatus (version: %s, revision: %s, revision-date: %s)", buildInfo.Version, buildInfo.Revision, buildInfo.Date)
```
I'm also thinking about removing it altogether since the revision date doesn't say anything about the actual build date the date can always be looked up in version history anyway if needed.